pkg/alert/alert.go

```diff
@@ -24,149 +24,44 @@ package alert
 import (
 	"fmt"
 
-	horizonapi "github.com/blackducksoftware/horizon/pkg/api"
-	"github.com/blackducksoftware/horizon/pkg/components"
+	"github.com/blackducksoftware/synopsys-operator/pkg/api"
+	v1 "github.com/blackducksoftware/synopsys-operator/pkg/api/alert/v1"
```

Comment:
instead of v1, you can have it as alertapi, so that in future, if we modify the crd version, the change will only be in import statements

Reply:
Made these changes

Reply:
I'll do the same thing for files in synopsysctl in a different PR